test/SourceKit/VariableType/ranged.swift

```diff
@@ -0,0 +1,8 @@
+let x = 1
+let y = "abc"
+let z: String = "def"
+var w = 4
+
+// RUN: %sourcekitd-test -req=collect-var-type -pos=2:1 -length=23 %s -- %s | %FileCheck %s
+// CHECK: (2:5, 2:6): String (explicit type: 0)
+// CHECK: (3:5, 3:6): String (explicit type: 1)
```

Comment:
In this case no, since we are testing ranged requests here.

Reply:
Ah, I didn't see that. Could you add the following lines
```
// CHECK-NOT: (1:5, 1:6)
```
and 
```
// CHECK-NOT: (4:5, 5:6)
```

At the moment, file check will pass even if these lines occur in the output because it will simply skip over lines that don’t match any `CHECK` line.